sqlglot/dialects/dialect.py

```diff
@@ -260,8 +260,8 @@ class Dialect(metaclass=_Dialect):
     If empty, the corresponding trie will be constructed off of `TIME_MAPPING`.
     """
 
-    ESCAPE_SEQUENCES: t.Dict[str, str] = {}
-    """Mapping of an unescaped escape sequence to the corresponding character."""
+    UNESCAPED_SEQUENCES: t.Dict[str, str] = {}
+    """Mapping of an unescaped sequence (`\\n`) to its escaped version (`\n`)."""
```

Comment:
Should this be "Mapping of an escaped sequence to it's unescaped version" ?

Reply:
Yeah thanks for catching